README.md

```diff
@@ -1,22 +1,144 @@
+
+</div>
+
+<br />
+
+# Getting Started
+
+This project was generated with React.Js & Electron, Follow below steps to get started.
+
+## Prerequisites
+
+Any Operating System will work !
+
+1. Node v. 12.x+
+
+2) [Tezster-CLI](https://docs.tezster.tech/tezster-cli)
+
+## Installation Process :
+
+- Setup your Node.js environment
+
+* Install and setup [Tezster-CLI](https://docs.tezster.tech/tezster-cli)
+
+## Node.js Installation Guide
+
+Run following commands to install Node.js LTS version
+
+    sudo apt-get update
+    sudo apt-get install curl
+    curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+    sudo apt-get install nodejs
+
+After installing verify and check the installed version.
+
+    node -v
+
+_Note: Please make sure you have the latest npm and node.js versions to be 12.x or above._
+
+## Install Tezster-CLI package
+
+[Tezster-CLI](https://docs.tezster.tech/tezster-cli) will create a sandboxed nodes in your local machine and you will be able to interact with it flawlessly with this GUI version.
+
+    sudo npm install -g tezster@latest
+
+Run Following to ensure the installed version of [Tezster-CLI](https://docs.tezster.tech/tezster-cli).
+
+    tezster --version
+
+To start the tester nodes on your machine run:
+
+    tezster start-nodes
+
+## Tezster-GUI App Setup
+
+Inorder to start using the app we clone this repo using the following command
+
+    git clone https://github.com/Tezsure/Tezster-GUI.git
+    cd Tezster-GUI/
+    sudo npm install
+
+‌
+
+Now to start the app in the `dev` environment run the following command :
+
+    npm run dev
 
 This starts the renderer process in hot-module-replacement mode and starts a webpack dev server that sends hot updates to the renderer process.
+
+_You are now ready to use the app or start the code development._
+
+## Building Packages for All Platforms
+
 To create the build package for different OS based environment run :
 
-```bash
-$ npm run package
-```
+    npm run package
+
+This will create a build package in the `release folder` of the root directory depending upon the OS you're working on.
 
-This will create a build package in the release folder of the root directory depending upon the OS you're working on. On linux systems it requires an additional dependency of wine package installer to create a build package.
 To create build package for other platform run the following command
 
-```bash
-$ npm run package-all
-```
+    npm run package-all
+
+On linux systems it requires an additional dependency of wine package installer to create a build package for windows environment.‌
+
+## Steps to install wine on linux
```

Comment:
This requires smaller font-size as it's a sub-header under 'Building packages for all platforms'